app/controllers/concerns/blacklight/catalog.rb

```diff
@@ -183,26 +185,6 @@ def render_document_export_format format_name
       render text: @response.documents.map { |x| x.export_as(format_name) if x.exports_as? format_name }.compact.join("\n"), layout: false
     end
 
-    # override this method to change the JSON response from #index 
```

Comment:
Does it make sense to do so?  Was anyone ever doing anything with this? Can we just add this as a breaking change in the release notes?